modules/runner/src/main/java/org/apache/ignite/internal/app/IgniteImpl.java

```diff
@@ -308,6 +311,12 @@ public class IgniteImpl implements Ignite {
     /** System views manager. */
     private final SystemViewManagerImpl systemViewManager;
 
+    /** Index builder. */
+    private final IndexBuilder indexBuilder;
```

Comment:
I think it would be nice to encapsulate some of these components into another umbrella component. IgniteImpl becomes too big. Not this time, of course

Reply:
Ok